client/src/js/modules/samples/views/proteinlist.js

```diff
@@ -4,107 +4,99 @@ define(['marionette', 'backgrid', 'views/table', 'views/filter',
   'templates/samples/proteinlist.html'], 
   function(Marionette, Backgrid, TableView, FilterView, ComponentTypes, table, Template) {
     
+    var ClickableRow = table.ClickableRow.extend({
+        event: 'proteins:view',
+        argument: 'PROTEINID',
+        cookie: true,
+    })
+        
+    return Marionette.LayoutView.extend({
+        className: 'content',
+        template: Template,
+        regions: { 'wrap': '.wrapper', type: '.type' },
     
-  var ClickableRow = table.ClickableRow.extend({
-    event: 'proteins:view',
-    argument: 'PROTEINID',
-    cookie: true,
-  })
+        clickableRow: ClickableRow,
+        
+        title: 'Protein',
+        url: 'protein',
     
-  return Marionette.LayoutView.extend({
-    className: 'content',
-    template: Template,
-    regions: { 'wrap': '.wrapper', type: '.type' },
-    // ui: {
-    //   add: 'a.add',
-    // },
-
-    clickableRow: ClickableRow,
-    showFilter: true,
```

Comment:
This is a duplicate but the duplicate is the wrong type so I am not sure what to do testing has releaved no change